images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -24,13 +24,13 @@ restore_pv_data
 cd ${APP_ROOT}
 bin/rake evm:deployment_status
 
+# Generate httpd certificate
+/usr/bin/generate_miq_server_cert.sh
```

Comment:
Nevermind it only generates the certs if they don't exist already, which they won't now that we are not saving them but we are not even checking them now that the httpd pod is providing the service attached to the route.

So this is fine for now as long as we take care of it in the future.